src/Symfony/Component/Form/Extension/Core/Type/ColorType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class ColorType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getParent()
+    {
+        return __NAMESPACE__.'\TextType';
```

Comment:
You should only do this on new classes. We usually avoid doing changes on existing code when it doesn't fix anything or doesn't really improve the situation, in order to avoid useless conflicts when the maintainers are merging lower branches to upper ones and interfering in git history.